src/firefly/java/edu/caltech/ipac/firefly/server/query/ibe/IbeQueryArtifact.java

```diff
@@ -229,6 +268,63 @@ private static RelatedData makeWiseRelatedDataItem(WorldPt wp, String sizeInDeg,
         return RelatedData.makeTabularRelatedData(params, dataKey, desc);
     }
 
+    public static  List<RelatedData> get2MassRelatedData(WorldPt wp, String sizeInDeg) {
+        List<RelatedData> rdList= new ArrayList<>();
+        rdList.add(get2MassRelatedDataItem(wp,sizeInDeg,"glint","glint_arti","2MASS Glints Artifacts"));
+        rdList.add(get2MassRelatedDataItem(wp,sizeInDeg,"pers","pers_arti","2MASS Persistence Artifacts"));
+        return rdList;
+    }
+
+    private File getWiseScanIdArtifact(final TableServerRequest req) throws IOException, DataAccessException {
```

Comment:
I don't think you are using this function, If you want to keep it then you should document why.